src/main/webapp/paginationScript.js

```diff
@@ -68,11 +70,16 @@ function displayCards(listAugment) {
         loadImage(resultsImageElement, cardToAppend.photoReference);
       }
 
+      let websiteButtonElement = locateWebsiteUrlElement(cardToAppend.card);
+      loadWebsiteUrl(websiteButtonElement, cardToAppend.placeId);
+
       resultsContent.appendChild(cardToAppend.card);
     }
   }
 }  
 
+const KEY = 'REDACTED';
```

Comment:
nit: it would be nice to keep all constants at the top of the file.

Reply:
fixed!